### PR TITLE
Norman init

### PR DESCRIPTION
**Pull request by normanchenn**

### CODE REVIEW

**Review by Chen-Prototypes — DISMISSED** [review]

File: include/board.h [hunk 1]
```diff
@@ -12,24 +12,22 @@ class Piece;
...
+    int fullmove_clock;
     bool verify_board();
-    void place_piece(
-        Color color, Coordinate square, PieceType type);
-    void remove_piece(Coordinate square);
     std::string deserialize();
 
 public:
     Board();
+    Board(int i);
```
Comment (PeteMango) [inline include/board.h:30]: the board constructor with (int i) should be merged into the default constructor

File: include/chess.h [hunk 2]
```diff
@@ -4,27 +4,33 @@
...
-    std::vector<Display> displays;
+    std::vector<std::unique_ptr<Display>> displays;
     void get_scores();
-    void notify_displays();
 
 public:
     Chess();
     void resign();
-    void setup_board(std::istream& in);
+    void setup_board(std::istream& in, bool& is_eof_given);
```
Comment (PeteMango) [inline include/chess.h:22]: well done! smart workaround!

File: src/display/textdisplay.cpp [hunk 3]
```diff
@@ -1,9 +1,68 @@
...
+        return;
+    }
+    std::shared_ptr<Game> g = this->chess->get_last_game();
+    std::cout << 1 << std::endl;
+    std::shared_ptr<Board> b = g->get_board();
+    std::cout << 2 << std::endl;
+    std::vector<std::vector<std::shared_ptr<Piece>>>& grid
+        = b->get_grid();
+    std::cout << 3 << std::endl;
+    for (int row = 7; row >= 0; row--) {
```
Comment (PeteMango) [inline src/display/textdisplay.cpp:43]: wrong, fix this

**Review by PeteMango — CHANGES_REQUESTED** [review]

**Review by PeteMango — APPROVED** [review]
lgtm

**Pull request merged**